src/main/java/org/folio/service/budget/BudgetService.java

```diff
@@ -81,17 +84,18 @@ private Future<Void> deleteAllocationTransactions(Budget budget, DBClient client
 
     client.getPgClient().execute(client.getConnection(), sql, reply -> {
       if (reply.failed()) {
-        logger.error("Allocation Transaction deletion by query {} failed", sql, reply.cause());
+        logger.error("deleteAllocationTransactions:: Allocation transaction deletion by query {} failed", sql, reply.cause());
```

Comment:
Lets include budgetId into this log as well

Reply:
added